build.sbt

```
@@ -32,7 +32,10 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
   val excludedPackages = Seq(
     "<empty>",
-    "Reverse.*",
+    "Reverse*",
+    "com.kenshoo.play.metrics*",
```

```suggestion
    "com.kenshoo.play.metrics.*",
```
The controller still shows up in coverage, needs the `.`